plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm

```diff
@@ -138,40 +138,8 @@ macro "Time-Lapse Color Coder" {
 }
 
 function makeLUTsArray() {
```

Comment:
How about removing this function entirely, and changing line 146 to:

```
lutA = getList("LUTs");
```

Reply:
Works for me. I was keeping it for historical purposes :)